src/unistd.rs

```diff
@@ -587,6 +588,38 @@ mod linux {
         Errno::result(res).map(drop)
     }
 
+    /// Sets the real, effective, and saved uid
```

Comment:
This is a real nit, but when rendered I think this will juxtapose the first line and "Further Reading", so it probably makes sense to add a period after the first line or do something like `([see setresuid(2)][...])`